zdm/parameters.py

```diff
@@ -112,7 +110,7 @@ class IGMParams(myDataClass):
         default=0.32,
         metadata={'help': 'F parameter in DM PDF for the Cosmic web'})
 
-# Host parameters
+# Host parameters -- host
 @dataclass
 class HostParams(myDataClass):
```

Comment:
I would have expected host parameters to include the log width too. What am I missing?